runtime/ftplugin/typst.vim

```diff
@@ -1,14 +1,43 @@
-" Vim filetype plugin
-" Language:	typst
-" Maintainer:	Riley Bruins <ribru17@gmail.com>
-" Last Change:	2024 May 19
+" Vim filetype plugin file
+" Language:    Typst
+" Maintainer:  Gregory Anders
+" Last Change: 2024-07-12
+" Based on:    https://github.com/kaarmu/typst.vim
 
 if exists('b:did_ftplugin')
   finish
 endif
 let b:did_ftplugin = 1
 
-setlocal comments=sO:*\ -,mO:*\ \ ,exO:*/,s1:/*,mb:*,ex:*/,://
+compiler typst
+
 setlocal commentstring=//\ %s
+setlocal comments=s1:/*,mb:*,ex:*/,://
+setlocal formatoptions+=croq
+setlocal suffixesadd=.typ
+
+if has('win32')
+    setlocal iskeyword=a-z,A-Z,48-57,_,-,128-167,224-235
+else
+    setlocal iskeyword=a-z,A-Z,48-57,_,-,192-255
+endif
```

Comment:
Why do we need different iskeyword settings for different systems? Is that for word motions or for the syntax keyword script in which case it should probably ba a :syn iskeyword command in the syntax script.

Reply:
The setting itself was changed in https://github.com/kaarmu/typst.vim/issues/77 but it looks like they just set it to the default value. So I'll delete these lines.